lib/src/room.dart

```diff
@@ -34,9 +34,15 @@ abstract class Room implements Listenable {
   /// If the service request has not yet been assigned, this will return `null`.
   String? get agentName;
 
+  /// Getter providing the [MessagingClient].
+  ///
+  /// If the application does not support messaging, the returned value will be null.
+  MessagingClient? get messagingClient;
+
   /// A broadcast stream of messages sent and received.
   ///
   /// If the application does not support messaging, this will throw an exception.
+  @Deprecated('This getter was moved into [MessagingClient].')
```

Comment:
Since this SDK is not at a `1.0.0` yet, I don't think you have to stress about backward compatibility. You can do something similar to https://github.com/aira/flutter_aira/pull/12, where you use a `!` to indicate that there's a breaking change in the PR and call it out in the release with `BREAKING` (e.g. https://github.com/aira/flutter_aira/releases/tag/0.0.9).

Reply:
where did you use a `!`?